pom.xml

```diff
@@ -499,12 +499,12 @@
       </plugin>
 
       <plugin>
-        <groupId>com.nickwongdev</groupId>
+        <groupId>dev.aspectj</groupId>
         <artifactId>aspectj-maven-plugin</artifactId>
-        <version>1.12.6</version>
+        <version>1.13.1</version>
         <configuration>
           <verbose>true</verbose>
-          <complianceLevel>11</complianceLevel>
+          <complianceLevel>17</complianceLevel>
```

Comment:
Please update also version of data-import-processing-core
It should be 4.1.0-SNAPSHOT

Reply:
Updated